Airports/EGPH/SMR/Labels.txt

```diff
@@ -108,3 +106,6 @@
 "210" N055.57.00.702 W003.20.54.574 standHold
```

Comment:
Stand 209 missing.
![default](https://user-images.githubusercontent.com/18355854/29998208-e7ce3dd8-902d-11e7-928f-1612f1a65b15.png)

Reply:
209 is never used real world, nor would it display correctly.